acceptance/tests/ticket_2455_on_solaris_init_provider_should_start_service_in_own_smf_contract.rb

```diff
@@ -0,0 +1,98 @@
+test_name "(PUP-2455) Service provider should start Solaris init service in its own SMF contract"
+
+step "Setup fixture service manifest on master"
+
+  testdir = master.tmpdir('solaris_services_in_own_smf_contract')
+
+  test_manifest = <<MANIFEST
+File {
+  ensure => directory,
+  mode => "0750",
+  owner => #{master.puppet['user']},
+  group => #{master.puppet['group']},
+}
+file { '#{testdir}': }
+file { '#{testdir}/environments': }
+file { '#{testdir}/environments/production': }
+file { '#{testdir}/environments/production/manifests': }
+file { '#{testdir}/environments/production/manifests/site.pp':
+  ensure  => file,
+  content => '
+    node default {
+      service { "vmware-tools":
```

Comment:
Agreed, a more universal daemon would be better. It looks like the few scripts that are in /etc/init.d in a default Solaris install are actually wrappers for SMF 'svcadm' commands. My first thought is to try to pick something from the OpenCSW packages that are installed by our full acceptance suite.